cetlvast/suites/compile/test_unbounded_variant_footprint_get_const.cpp

```diff
@@ -8,19 +8,10 @@
 ///
 
 #include "cetl/unbounded_variant.hpp"
+#include "cetlvast/helpers_rtti.hpp"
 
 #include <cstdint>
 
-namespace cetl
-{
-template <>
-constexpr type_id type_id_value<uint8_t>{};
-
-template <>
-constexpr type_id type_id_value<uint16_t>{};
-
-}  // namespace cetl
```

Comment:
Note that in examples I deliberately don't use this new `cetlvast/helpers_rtti.hpp` and I left such definitions as is. Rationale: so that examples will be more self-sufficient and explicitly show how to declare a type id.